packages/schema/src/config/experimental.ts

```diff
@@ -589,5 +589,10 @@ export default defineResolvers({
         return typeof val === 'boolean' ? val : true
       },
     },
+
+    /**
+     * Whether Nuxt should stop if a Nuxt module is incompatible.
+     */
+    enforceModuleCompatibility: false,
```

Comment:
Maybe can this be moved into `future` instead and be enabled by default for v4 ?

Reply:
Let's enable it by default for v4 (in the experimental namespace), and document in the migration guide. This can be done in a patch release if necessary.